### PR TITLE
Update guidelines

### PR DESCRIPTION
**Pull request by cyyynthia**
Yes I made some changes, to make the guidelines a bit better. Those changes include:
 - Some reformatting
 - Add a rule about NSFW plugins
 - Clarifications about how the Powercord Staff will handle your plugins

Since there are probably some broken baguette english as always, feel free to correct me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,77 +1,98 @@
...
-Gateway. Performing such actions may get user's account flagged for selfbotting or modding and banned for doing so.
+Discord internally has mostly everything you'll need to retrieve data without directly calling the API or connecting to
+the Gateway. Performing such actions may get user's account flagged for selfbotting or modding and banned for doing so.
 
-## 11. Powercord Developers will never directly push in your plugin's repo
+### 11. Limit NSFW plugins
 
-Even if we have full access by being administrators of the organization, we'll never push to a plugin's repo unless
-we're invited to do so. We'll always prefer forking and making a PR.
+We're not against plugins providing NSFW features, but they should be properly labeled and handled. If you plugin can
```
Comment (ghostlydilemma) [inline README.md]: If **your** plugin

File: README.md [hunk 2]
```diff
@@ -1,77 +1,98 @@
 # Powercord Community Guidelines
 
-## 1. Properly structure your plugin's manifest
+Those guidelines are set to ensure Powercord's ecosystem stays as awesome as possible, and to avoid malicious plugins,
```
Comment (ghost) [inline README.md]: "These" would be used better here than "Those"

File: README.md [hunk 3]
```diff
@@ -1,77 +1,98 @@
...
-## 5. Do not write harmful plugins
+### 5. Do not write harmful plugins
 
-Harmful plugins that abuse Powercord APIs or Discord internals to do malicious things like spambots, keyloggers
-or anything else are strictly prohibited.
+Harmful plugins that abuse Powercord APIs or Discord internals to do malicious things (including but not limited to
+spambots, keyloggers, scrapers, ...) are strictly prohibited.
 
-## 6. Do not have bad privacy practices
+### 6. Respect users privacy
```
Comment (Yumiro) [inline README.md]: Respect the user's privacy would sound better imo

File: README.md [hunk 4]
```diff
@@ -1,77 +1,98 @@
...
 
 Collecting user data using Powercord without consent is forbidden. All features that require the collection of user data
-should be opt-in, and if your plugin is based on user data please specify it in `permissions` manifest key.
+should be opt-in (if possible), and properly reflect data collection with the `permissions` manifest key.
+
+### 7. Limit use of external APIs
 
-## 7. Limit use of external APIs
+External APIs, even open source, may secretely do malicious actions, as returning malicious data, logging your IP
```
Comment (ghost) [inline README.md]: "secretly" is the correct form, not "secretely"

File: README.md (hunk 3 above)
Comment (davidcralph) [reply README.md]: user's or user's (not sure which)

File: README.md [hunk 5]
```diff
@@ -1,77 +1,98 @@
...
 
-## 7. Limit use of external APIs
+External APIs, even open source, may secretly do malicious actions, as returning malicious data, logging your IP
+address and more. If you have to use your API or another one that is not provided by a well-known entity (like Spotify
+API), please add the `ext_api` permission to your manifest.
 
-External APIs, even open source, may secretely do malicious actions, as returning malicious data, logging your IP address
-and more. If you have to use your API or another one that is not provided by a well-known entity (like Spotify API),
-please add the `ext_api` permission to your manifest.
+**Note**: If your plugin requires the use of an external custom-built backend, Powercord is ready to hook you up with
```
Comment (davidcralph) [inline README.md:53]: and a domain

File: README.md [hunk 6]
```diff
@@ -1,77 +1,98 @@
...
-Gateway. Performing such actions may get user's account flagged for selfbotting or modding and banned for doing so.
+Discord internally has mostly everything you'll need to retrieve data without directly calling the API or connecting to
+the Gateway. Performing such actions may get user's account flagged for selfbotting or modding and banned for doing so.
 
-## 11. Powercord Developers will never directly push in your plugin's repo
+### 11. Limit NSFW plugins
 
-Even if we have full access by being administrators of the organization, we'll never push to a plugin's repo unless
-we're invited to do so. We'll always prefer forking and making a PR.
+We're not against plugins providing NSFW features, but they should be properly labeled and handled. If your plugin can
```
Comment (davidcralph) [inline README.md:74]: are apple bottom jeans and boots with the fur nsfw?

File: README.md [hunk 7]
```diff
@@ -1,77 +1,98 @@
...
+provide NSFW but it isn't the main purpose, **NSFW features must be turned off by default**. If the main feature of
+the plugin is based on NSFW, please specify it in your manifest through the `nsfw` key, so we can label it as such.
+
+## How we handle plugins
 
-<!--
```
Comment (davidcralph) [inline README.md:74]: thank you

File: README.md [hunk 8]
```diff
@@ -1,77 +1,98 @@
...
-| description          | Short description of what the plugin is doing.                                                                   |          |
-| author               | Name(s) of the author(s).                                                                                        |          |
-| license              | License the plugin is released under.                                                                            |          |
-| permissions          | List of permission the plugin is asking (`keypresses`, `use_eud`, `ext_api`).                                    | x        |
-| dependencies         | List of plugin's dependencies.                                                                                   | x        |
-| optionalDependencies | List of plugin's optional dependencies. Will only affect load order to ensure the plugin is loaded before yours. | x        |
-| appMode              | Where the plugin should be injected. Can be `app`, `overlay`, or `both` (default to `app`).                      | x        |
-
-## 2. Follow Discord and other services ToS
+### 2. Follow Discord and other services ToS
```
Comment (davidcralph) [inline README.md]: services' (?)

Comment (cyyynthia) [reply README.md]: Then it'd also apply do "Discord" right? "Follow Discord's and other services' ToS"

File: README.md (hunk 6 above)
Comment (cyyynthia) [reply README.md:74]: 😩

File: README.md (hunk 8 above)
Comment (davidcralph) [reply README.md]: that could work

**Review by davidcralph — APPROVED** [review]

**Review by tilda — CHANGES_REQUESTED** [review]
"user's" sounds a bit broken to me, change to "user" possibly?

**Comment by aetheryx** [comment]
lgtm

**Comment by ohlookitsderpy** [comment]
please merge

**Pull request merged**